osu.Game.Rulesets.Osu/Difficulty/Evaluators/FlashlightEvaluator.cs

```diff
@@ -73,6 +79,30 @@ public static double EvaluateDifficultyOf(DifficultyHitObject current, bool hidd
             if (hidden)
                 result *= 1.0 + hidden_bonus;
 
+            double sliderBonus = 0.0;
+
+            if (osuCurrent.BaseObject is Slider)
+            {
+                Debug.Assert(osuCurrent.TravelTime > 0);
+
+                Slider osuSlider = (Slider)(osuCurrent.BaseObject);
```

Comment:
You can save writing a line initializing `osuSlider` if you just do it in the if statement instead:
```csharp
if (osuCurrent.BaseObject is Slider osuSlider)
```

can do the same in `AimEvaluator.cs`

Reply:
Fixed